components/datetime/README.md

```diff
@@ -52,14 +52,14 @@ let dtf = DateTimeFormatter::try_new_with_buffer_provider(&provider, &locale.int
 let typed_dtf = TypedDateTimeFormatter::<Gregorian>::try_new_with_buffer_provider(&provider, &locale!("en").into(), options)
     .expect("Failed to create TypedDateTimeFormatter instance.");
 
-let typed_date = parse_gregorian_from_str("2020-09-12T12:35:00").expect("Failed to parse date.");
+let typed_date = DateTime::new_gregorian_datetime(2020, 9, 12, 12, 34, 28).unwrap();
```

Comment:
Nit: Favor `new_iso_datetime` in docs examples without arithmetic (see other thread). This one is tricky though since you need to use it for both TypedDTF and AnyDTF.